### ISSUE TITLE
Missing download content

### ISSUE THREAD

**Issue by jjimijjames** [kzahel/jstorrent-legacy-app#146]
Hello, I have installed js torrent using the terminal on my Chromebook and adding the chrome web server code to the application folder and everything seemed to work fine. I am downloading magnet links by pasting the magnet URL into js torrent. Problem is that with every thing I download, the gui says everything is downloaded and at what speed and etc. but when I go to the download directory there are always missing files. So far have not been able to get one complete file. THey are either missing completely where the gui says they've been downloaded, or they are there but are corrupt and unusable. Any feedback would be appreciated.

thanks,

james

**Comment by kzahel** [comment]
Super strange. Can you try to see if you have the same problem with the webstore version? (here is the free trial: https://chrome.google.com/webstore/detail/jstorrent-lite/abmohcnlldaiaodkpacnldcdnjjgldfh)

**Comment by jjimijjames** [comment]
Ok so I did not have any problem with this free trial app and I was able to
download and subsequently locate the expected files. One thing that caught
my attention however was that upon launching the app initially I found a
path to configure js torrent to handle magnet links. once I did that I no
longer had to copy and paste the magnet url, I could just click on it a and
it would open in js torrent. So I'm thinking perhaps when I installed
developer js torrent I never completed that configuration and I think
that's probably whats screwing me up.

THanks for your help

James W. Levergood
226.345.6058
